runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -348,6 +410,7 @@ TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node)
                               "Unexpected broadcast type in node %p\n", node);
 
                _aliasTable[methodRefNum]._objectType =(broadcastType == MODE_BROADCAST) ? Vector : Mask;
+               _nodeTable[nodeIndex]._objectType =(broadcastType == MODE_BROADCAST) ? Vector : Mask;
```

Comment:
Please add a space between `=` and `(broadcastType`.

Reply:
done

Reply:
Would you do the same for the previous line for `_aliasTable`, although it is existing code, please?  Thanks.

Reply:
I realized this code is not needed. Replaced with correct, simpler code. Basically, we need to determine the object type (Mask or Vector) from the first parameter of this intrinsic.